packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,126 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 5px);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-small-horizontal-inset, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
+    font-size: var(--notification-dialog-body-font-size, 1.25rem);
+  }
+
+  // This flex container is needed for IE 10 / IE 11 to render correctly
+  .notification-dialog-inner-wrapper {
+    display: flex;
+    flex-direction: row;
+    max-height: 80vh;
+  }
+
+  .notification-dialog-container {
+    display: flex;
+    flex-direction: column;
+    max-height: inherit; // `inherit` value needed for IE10 to scroll modal-body when overflowed
+    width: 100%; // Needed for IE to wrap content correctly in flexbox
+  }
+
+  .header-body {
+    background-color: var(--notification-dialog-header-background-color, #f9f9f9);
+    border-bottom: var(--notification-dialog-header-border-bottom, 0.0625rem solid #dedfe0);
+    font-size: var(--notification-dialog-header-font-size, 1.125rem);
+    padding: var(--notification-dialog-body-padding, 0.8rem);
```

Comment:
need to update this one `--notification-dialog-body-padding` (also used on line 21) to `--notification-dialog-header-padding` for here.